docs/Gemfile

```diff
@@ -3,8 +3,8 @@ source 'https://rubygems.org'
 
 # Middleman
 gem 'middleman', '~>4.2'
-gem 'middleman-autoprefixer', '~> 2.7'
-gem 'middleman-sprockets', '~> 4.1'
+gem 'middleman-autoprefixer', '~> 3.0'
+gem 'middleman-sprockets', '~> 4.0'
```

Comment:
Not blocking - Ideally we only fix the gems we need to and remove all these version constraints, thus allowing bunder to determine which set of dependencies are allowed (and easily update them). 

If we did this and lock `gem 'middleman-sprockets', '~> 4.0'` then autoprefixer could have updated without intervention on our end.